examples/apps/generic_client/eth_generic_client.py

```diff
@@ -581,6 +564,7 @@ def Main(args=None):
             exit(1)
 
     # Submit work order
+    jrpc_req_id = random.randint(0, 100000)
```

Comment:
I think we can use either this approach or follow existing approach (line 571) which increments jrpc_req_id for each request. We don't need both.

Reply:
What is this field used for? If this should be a unique ID then I recommend using UUID.

Reply:
Good suggestion @dcmiddle . This would be the way to go. 

It is a [field](https://www.jsonrpc.org/specification#request_object) used to verify the context of a response receive after posting a JRPC request. As of now, it is not being verified in [Avalon's sdk](https://github.com/hyperledger/avalon/blob/master/sdk/avalon_sdk/http_client/http_jrpc_client.py#L56) as the communication is over http and only a single call is included in each call.

When there is significant traffic, multi-call JRPC requests might be possible in a single HTTP request. This field would play a role there to map requests to responses. A shift from HTTP(synchronous request-response) would also require proper handling of this field.

Reply:
Added to FAQ. PR #309